File: code/python/lab9.py
Am,Fm,Ko,Gain = 8,4,10,10    #all parameters ans with Khz so not include 10^3 part
print(f"A. The peak frequency deviation is Deltaf equal to {Am*Gain} KHz\n")
print(f"B. The modulating index is Betaf equal to {int((Am*Gain)/Fm)}\n")
print(f"C. The phase modulating index is Betaf equal to  {Am*Ko} radians") 

File: code/matlab/lab9.m
clc;
clear all;
close all
Fm = 4; %unit in kHz
mt_max_value = 8; %unit in V
womeca = 10; %unit in kHz/V

peak_deviation_delta_f = mt_max_value*womeca;
fprintf('a) Peak deviation delta_f: %d KHz\n',peak_deviation_delta_f)

fre_modu_index_Bf=(peak_deviation_delta_f/Fm);
fprintf('b) Frequency modulation index Bf: %d\n',fre_modu_index_Bf)

phase_modulation_index = womeca*mt_max_value;
fprintf('c) Phase modulation index: %d radians\n',phase_modulation_index)